### PR TITLE
BibID Updater Fixes

### PR DESCRIPTION
**Pull request by tpendragon**
I found that the cursor used in the find_by_property query forces a
transaction, meaning any changes weren't reflected in the edit forms of
the UI, and resulted in Solr being in a different state than postgres.

There's plenty of memory to load it in, and it only takes a minute or
so.

### CODE REVIEW

File: app/services/bibid_updater.rb [hunk 1]
```diff
@@ -21,29 +21,32 @@ def update
   private
 
     def change_set_persister
-      @change_set_persister ||= ChangeSetPersister.new(metadata_adapter: Valkyrie.config.metadata_adapter,
-                                                       storage_adapter: Valkyrie.config.storage_adapter)
+      @change_set_persister ||= ScannedResourcesController.change_set_persister
```
Comment (hackartisan) [inline app/services/bibid_updater.rb:24]: You like this better? I think it's fine in a temporary service like this but I don't know that I'd want it sprinkled about.

**Review by hackartisan — APPROVED** [review]

File: app/services/bibid_updater.rb (hunk 1 above)
Comment (tpendragon) [reply app/services/bibid_updater.rb:24]: The previous one was actually wrong. If we started doing this for real I think `ChangeSetPersister.default` would be good.

Comment (hackartisan) [reply app/services/bibid_updater.rb:24]: 👍🏻 something like that would be good.

Comment (tpendragon) [reply app/services/bibid_updater.rb:24]: Created https://github.com/pulibrary/figgy/issues/4629

**Pull request merged**